packages/fast-stable-stringify/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "@solana/fast-stable-stringify",
+    "version": "1.0.0",
```

Comment:
I know this is a brand-new package, but let's make this `2.0.0-preview.2` like the others!